### PR TITLE
Refactor: Draft Invoice move to CustomerViewSet

### PR DESCRIPTION
**Pull request by Aashish-Upadhyay-101**

### CODE REVIEW

**Comment by mnida** [comment]
After review, merge this into a new branch so tests can run correctly

File: frontend/src/components/base/CopytoClipboard.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React, { useEffect, useState, useRef } from "react";
 import { CheckCircleOutlined, CopyOutlined } from "@ant-design/icons";
 import "./CopytoClipboard.css";
 import { Tooltip } from "antd";
-import Prism from "prismjs";
+// import Prism from "prismjs";
```
Comment (diego-escobedo) [inline frontend/src/components/base/CopytoClipboard.tsx:6]: Why was this commented out?

File: backend/api/views.py [hunk 2]
```diff
@@ -302,6 +304,52 @@ def archive(self, request, customer_id=None):
         CustomerDeleteResponseSerializer().validate(return_data)
         return Response(return_data, status=status.HTTP_200_OK)
 
+    @extend_schema(
```
Comment (diego-escobedo) [inline backend/api/views.py:307]: Also need to get rid of the other method.

**Review by diego-escobedo — CHANGES_REQUESTED** [review]
Need to see the path for draft invoice changed in the draft invoices tests, plus need to get rid of the old method. Additionally, a loom with the working draft invoice functionality (navigating to customer > subscriptions page in the frontend) would be great

File: frontend/src/components/base/CopytoClipboard.tsx (hunk 1 above)
Comment (Aashish-Upadhyay-101) [reply frontend/src/components/base/CopytoClipboard.tsx:6]: this is throwing me some frontend issue so i decided to comment it out !

File: backend/api/views.py (hunk 2 above)
Comment (Aashish-Upadhyay-101) [reply backend/api/views.py:307]: you mean the old draft_invoice?

**Pull request merged**